qualcoder/qualcoder.py

```diff
@@ -715,7 +761,8 @@ def clean_dialog_refs(self):
 
 @click.command()
 @click.option('-p','--project-path')
-def main(project_path):
+@click.option('-v','--view',is_flag=True)
```

Comment:
I asm not familiar with click or he need for it here.

Reply:
exactly, specially for experts and for development this can be very handy